index.js

```diff
@@ -99,238 +105,323 @@ const defaultProps = {
     touchableActiveOpacity:         0.2,
     sectionTextStyle:               {},
     selectedItemTextStyle:          {},
-    cancelContainerStyle:           {},
-    cancelStyle:                    {},
-    cancelTextStyle:                {},
+    finishContainerStyle:           {},
+    finishStyle:                    {},
+    finishTextStyle:                {},
     overlayStyle:                   {},
     initValueTextStyle:             {},
-    cancelText:                     'cancel',
+    finishText:                     "Done",
     disabled:                       false,
-    supportedOrientations:          ['portrait', 'landscape'],
-    keyboardShouldPersistTaps:      'always',
+    supportedOrientations:          ["portrait", "landscape"],
+    keyboardShouldPersistTaps:      "always",
     backdropPressToClose:           false,
     openButtonContainerAccessible:  false,
     listItemAccessible:             false,
-    cancelButtonAccessible:         false,
+    finishButtonAccessible:         false,
     scrollViewAccessible:           false,
     scrollViewAccessibilityLabel:   undefined,
-    cancelButtonAccessibilityLabel: undefined,
+    finishButtonAccessibilityLabel: undefined,
     passThruProps:                  {},
     selectTextPassThruProps:        {},
     optionTextPassThruProps:        {},
-    modalOpenerHitSlop:             {top: 0, bottom: 0, left: 0, right: 0},
+    modalOpenerHitSlop:             { top: 0, bottom: 0, left: 0, right: 0 },
     customSelector:                 undefined,
-    selectedKey:                    '',
+    initSelectedKeys:               [],
+    multiple:                       false,
+    renderCheckbox:                 undefined,
```

Comment:
The Readme says that `renderCheckbox` is required (not optional).  However it's not needed for single select mode, only multi select.  Please provide a default empty method for renderCheckbox `() => {}`, update the Readme to reflect that the prop is optional, and in the description note that the user should provide a render method if they want checkboxes to display.

Reply:
Handled in 1d4d858.